content/hi/docs/index.md

```diff
@@ -0,0 +1,14 @@
+---
+approvers:
+- chenopis
+- abiogenesis-now
```

Comment:
```suggestion
```

Reply:
We are not sure if they are approvers for Hindi version of this page too. We can safely omit `approvers:` here.

Reply:
So approvers are left out now, right?

Reply:
Yep its not necessary to have `approvers:` on every page in localization. With Hindi localization, we are not having approvers for pages, so we can omit this section like we did on other pages